### DISCUSSION TITLE
%E5%8F%AF%E5%A1%91%E6%80%A7%E8%AE%B0%E5%BF%86/

### DISCUSSION THREAD

**Discussion by giscus** [ThatCoders/thatcoders.github.io#13] [Comment]
# 2022/07/14/Ninth%20Art/GalGame/%E5%8F%AF%E5%A1%91%E6%80%A7%E8%AE%B0%E5%BF%86/

作品背景为智能机器人普及后的近未来，故事的主线为描绘智能机器人的恋爱故事。

https://blog.thatcoder.cn/2022/07/14/Ninth%20Art/GalGame/%E5%8F%AF%E5%A1%91%E6%80%A7%E8%AE%B0%E5%BF%86/

**Comment by weekdaycare** [comment]
喔喔，看起来效果不错，话说是可塑性记忆诶，太催泪了，当时看一半都不敢看下去了🥹

**Reply by tovarsh** [reply]
其实，我也看不下去😂
